src/main/java/uk/gov/companieshouse/search/api/service/search/impl/dissolved/DissolvedSearchIndexService.java

```diff
@@ -1,5 +1,16 @@
 package uk.gov.companieshouse.search.api.service.search.impl.dissolved;
 
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.companieshouse.search.api.exception.SearchException;
+import uk.gov.companieshouse.search.api.logging.LoggingUtils;
+import uk.gov.companieshouse.search.api.model.SearchResults;
+import uk.gov.companieshouse.search.api.model.esdatamodel.Company;
+import uk.gov.companieshouse.search.api.model.response.ResponseObject;
+import uk.gov.companieshouse.search.api.model.response.ResponseStatus;
+
+import java.util.Map;
```

Comment:
static imports are supposed to be before non-static imports as specified https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing